app/src/main/java/com/github/se/project/ui/lesson/AddLesson.kt

```diff
@@ -27,7 +28,7 @@ fun AddLessonScreen(
     navigationActions: NavigationActions,
     listProfilesViewModel: ListProfilesViewModel,
     lessonViewModel: LessonViewModel,
-    onMapReadyChange: (Boolean) -> Unit
+    onMapReadyChange: (Boolean) -> Unit = {}
```

Comment:
This is good and help for the testing when the field is not used. However, I do not think you were assign to this bug fix, if I remember correctly it was in the task of Maik. I do not think it is a big problem since it is very small change but be careful in the future to avoid doing task of others.